travis/test_server.py

```diff
@@ -296,11 +304,33 @@ def main(argv=None):
     counted_errors = 0
     for to_test in to_test_list:
         print("\nTesting %s:" % to_test)
-        subprocess.call(["createdb", "-T", dbtemplate, database])
+        db_odoo_created = False
+        try:
+            db_odoo_created = subprocess.call(
+                ["createdb", "-T", dbtemplate, database])
+        except subprocess.CalledProcessError:
+            db_odoo_created = True
         for command, check_loaded in commands:
-            command[-1] = to_test
-            # Run test command; unbuffer keeps output colors
-            command_call = ["unbuffer"] + command
+            if db_odoo_created and instance_alive:
```

Comment:
You are right!
This section use a instance alive (a odoo server running eternally), then I don't knew how to test it easily.
We can think in a new way of test it.

By now we are testing this section in next [runbot-addons#71](https://github.com/OCA/runbot-addons/pull/71/files#diff-8cc021a18c100ed3cee5c1d675c8e723R91)
With next build: https://travis-ci.org/OCA/runbot-addons/jobs/92647163

It the first module of runbot with tests (The bad new is that don't increase the coverage of MQT) but you can give me a change of think in a way to test it here.